### ISSUE TITLE
Key configuration

### ISSUE THREAD

**Issue by niktor76** [KennethSundqvist/key-jump-browser-extension#59]
Hello!
Is it possible to change the key shortcuts? I would like to switch . and ,

**Comment by olivierdagenais** [comment]
I found the "Extension options" dialog:
![image](https://user-images.githubusercontent.com/297515/112325571-30fa0800-8c8a-11eb-8d74-3946222f1113.png)
...but:
1. The CSS defined for it hides anything you type:
    ```
    /*
    	Transparent color hides the caret in the input.
    	The shortcut text is shown using the placeholder property.
    */
    color: transparent;
    ```
    ...making it very difficult to see that you did type anything (my screenshot was _after_ I had typed in new values).
2. Neither of the values are saved.

**Comment by olivierdagenais** [comment]
I opened the Google Chrome DevTools Console after opening the Options and I see this:

```
options.js:120 Uncaught EvalError: Refused to evaluate a string as JavaScript because 'unsafe-eval' is not an allowed source of script in the following Content Security Policy directive: "script-src 'self' blob: filesystem:".

    at Object../src/options.js (options.js:120)
    at __webpack_require__ (options.js:20)
    at options.js:84
    at options.js:87
```

**Comment by olivierdagenais** [comment]
According to microsoft/Mezzurite-DevTools#27, it looks like this could be fixed by adding the line `devtool: 'source-map',` to `web-ext-config.js` (as they did in microsoft/Mezzurite-DevTools#40)

**Comment by olivierdagenais** [comment]
What do you think, @KennethSundqvist ?  Can you give that a try?  I'm not an extension developer but I would greatly appreciate configuring the keys for this extension!

**Comment by sergeevabc** [comment]
Aggrrhh, How do I remove/reset a shortcut? Pressing Delete or Backspace or even Esc binds that key instead of clearing the field.

**Comment by KennethSundqvist** [comment]
Hi @niktor76 and @olivierdagenais.

Thank you for reporting and following up on this. And I'm sorry for the late response.

These bugs are fixed in version 5.3.1 which was released yesterday.

**Comment by KennethSundqvist** [comment]
Hi @sergeevabc.

There is currently no way to remove or reset a shortcut in this extension. The shortcuts are essential to using the extension, so it doesn't make sense to me to have it installed without any shortcuts.

Or are you maybe only using one of the shortcuts?

**Comment by KennethSundqvist** [comment]
Closing this since the original topic has been resolved in version 5.3.1.
